spec/classes.md

```diff
@@ -2920,6 +2927,56 @@ public class ReadOnlyPoint
 Notice that the assignments to the readonly field are legal, because they occur within the constructor.
 
 
+The following example:
+```csharp
+// No 'field' symbol in scope.
+public class Point
+{
+	public int X { get; set; }
+	public int Y { get; set; }
+}
+```
+is equivalent to the following declaration:
+```csharp
+// No 'field' symbol in scope.
+public class Point
+{
+	public int X { get { return field; } set { field = value; } }
+	public int Y { get { return field; } set { field = value; } }
+}
+```
+which is equivalent to:
+```csharp
+// No 'field' symbol in scope.
+public class Point
+{
+    private int __x;
+    private int __y;
+    public int X { get { return __x; } set { __x = value; } }
+    public int Y { get { return __y; } set { __y = value; } }
+}
+```
+
+The following example:
+```csharp
+// No 'field' symbol in scope.
+public class LazyInit
+{
+	public string Value => field ??= ComputeValue();
```

Comment:
nit: this indent is an eye sore. Also below (line 2975)

Reply:
yup.  all whitespace was fixed.